src/Arcus.Observability.Telemetry.Core/Logging/DependencyLogEntry.cs

```diff
@@ -49,6 +49,7 @@ public class DependencyLogEntry
             StartTime = startTime.ToString(FormatSpecifiers.InvariantTimestampFormat);
             Duration = duration;
             Context = context;
+            Context[ContextProperties.General.TelemetryType] = TelemetryType.Dependency;
```

Comment:
Added this, not for functionality, but because it seemed something good to have, since when we format to JSON, there's otherwise no single property that can be used to determine which type it is.
But again, it's not being used in our functionality. I'll remove this if you think this is unnecessary.

Reply:
I currently have no opinion on that. :)  If I see the result, I can maybe form an opinion :)

Reply:
This is fine for me, as long as the Application Insights sink removes this for that telemetry, other sinks can have this without a problem imo